lms/djangoapps/grades/signals/handlers.py

```diff
@@ -244,6 +247,13 @@ def recalculate_course_grade(sender, course, course_structure, user, **kwargs):
     CourseGradeFactory().update(user, course=course, course_structure=course_structure)
 
 
+@receiver(ENROLLMENT_TRACK_UPDATED)
+@receiver(COHORT_MEMBERSHIP_UPDATED)
+def recalculate_course_grade_on_user_partition_change(sender, user, course_key, **kwargs):
```

Comment:
I know _exactly_ what this function does!

Reply:
Ok. gotcha.  didn't notice the `force_update_subsections` param.
In that case, can you update the names of these 2 methods so it's clear what the distinction is in these differences in their intention?
Also, as mentioned elsewhere, I wouldn't use the term user-partition at this grades layer - it's an abstraction violation since user-partitions are 2 layers down from this grades-integration with membership changes.

Reply:
For sure.